__tests__/Ship.test.js

```diff
@@ -0,0 +1,53 @@
+/* globals describe it expect */
+const Ship = require('../src/Ship.js');
+
+describe('Ship', () => {
+    describe('with ports and an itinerary', () => {
+        let ship;
+        let dover;
+        let calais;
+        let itinerary;
+        beforeEach(() => {
+            dover = {
+                addShip: jest.fn(),
+                removeShip: jest.fn(),
+                portName: 'Dover',
+                ships: []
+            };
+            calais = {
+                addShip: jest.fn(),
+                removeShip: jest.fn(),
+                portName: 'Calais',
+                ships: []
+            };
+            itinerary = {
+                ports: [dover, calais]
+            };
+            ship = new Ship(itinerary);
+        });
```

Comment:
It would be good to have a clear line between each of the `it` blocks here to make it a bit easier to read. This applies to all the test files